Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/GoogleAPIVersion.kt

```diff
@@ -0,0 +1,35 @@
+package de.rki.coronawarnapp.util
+
+import com.google.android.gms.common.api.ApiException
+import com.google.android.gms.common.api.CommonStatusCodes
+import de.rki.coronawarnapp.nearby.InternalExposureNotificationClient
+import kotlin.math.abs
+
+class GoogleAPIVersion {
```

Comment:
Maybe make this injectable. `@Inject constructor()`. The class is stateless, but can be recreated, so we can also tag the class `@Reusable`.